buildpipeline/DotNet-CoreFx-Trusted-OSX.json

```diff
@@ -79,7 +79,9 @@
       }
     },
     {
-      "environment": {},
+      "environment": {
+        "PACKAGEVERSIONPROPSURL": "$(PB_PackageVersionPropsUrl)"
```

Comment:
Did this work?

Reply:
Yep!  I confirmed with VSTS that this is the way to do it.